### DISCUSSION TITLE
Unable to run the simulator

### DISCUSSION THREAD

**Discussion by codeness0** [Fira-Autonomous-Cars/FIRA-Autonomous-Cars-Simulator#1] [Q&A] [answered]
Hello, 

I installed the simulator following the instruction, and the when I run the following command, It couldn't find the package.
**Command:**
`roslaunch avisengine_environment track_race_simple.launch`

**Output**
``` 
RLException: [track_race_simple.launch] is neither a launch file in package [avisengine_environment] nor is [avisengine_environment] a launch file name 
The traceback for the exception was written to the log file

**Comment by AmirmohammadZarif** [comment, marked as answer]
This is because of the `.bashrc` file, you need to source the `/devel/setup.bash` of your workspace in order to get it to work. You can add it to the .bashrc so it doesn't require running the `source` command, every time you open a new terminal.
```
echo 'source ~/catkin_ws/devel/setup.bash' >> ~/.bashrc
```
You can also use `nano` to add it to the end of the file.

**Reply by codeness0** [reply]
Thanks.
Ran the command, opened another terminal and it worked.
